console/console-init/ui/src/modules/iot-certificates/components/IoTCertificateList/IoTCertificateList.tsx

```diff
@@ -22,10 +27,18 @@ export interface IIoTCertificate {
 
 export interface IIoTCertificateListProps {
   certificates: IIoTCertificate[];
+  onSave: (certificate: IIoTCertificate) => void;
```

Comment:
use interface IIoTCertificateProps and extends these properties. don't need to redeclare.